onos-config/values.yaml

```diff
@@ -108,6 +117,12 @@ logging:
       output:
         stdout:
           sink: stdout
+    grpc:
```

Comment:
We should not add it here. It should be separate. We use a specific format for loggers

Reply:
done